README.md

```diff
@@ -186,28 +186,6 @@ See [dvt.obol.tech](https://dvt.obol.tech/) for the latest status of the Obol Ne
 > at least 3 finalised epochs before starting the charon cluster,
 > otherwise your validator could be slashed.
 
-## Run prysm VCs in a Distributed Validator Cluster
```

Comment:
Removing prysm as the current setup doesn't work and alternate approach needs to be implemented which would take some time

https://discord.com/channels/930779837218562078/1111645176394424410/1113504800789778644